src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -72,7 +73,7 @@ protected function formMultipleElements(FieldItemListInterface $items, array &$f
     $parents = $form['#parents'];
 
     $target_type = $this->fieldDefinition->getTargetEntityTypeId();
-    $user_groups = Og::getEntityGroups(\Drupal::currentUser()->getAccount());
+    $user_groups = Og::getEntityGroups(User::load(\Drupal::currentUser()->id()));
```

Comment:
I'm not sure if doing a `User::load()` using the uid from the user session is the best way of doing this. `AccountInterface` doesn't provide a method to load the full entity.